_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,266 @@
+		document.getElementById('error').innerText = ''
+		var age = document.getElementById('age').value
+		if (!age || isNaN(age) || age < 1) {
+			document.getElementById('error').innerText = 'Please fill the field correctly.'
+		}
+		var name = document.getElementById('name').value
+		if (!name) {
+			document.getElementById('error').innerText = 'Please fill the field correctly.'
+		}
+	}
+</script>
+
+<form>
+	<div id="error"></div>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<label for="name">Name</label>
+	<input type="text" id="name" />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+#### Failed Example 3
+
+The `input` elements has a [form field error indicator][] but its message does not describe the cause of the error.
+
+```html
+<script>
+	function processForm() {
+		document.getElementById('error').innerText = ''
+		var age = document.getElementById('age').value
+		if (!age || isNaN(age) || age < 1) {
+			document.getElementById('error').innerText = 'Please enter a correct age.'
+		}
+	}
+</script>
+
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<span id="error"></span><br />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+#### Failed Example 4
+
+The `input` element has a [form field error indicator][] that identifies it (by referencing its label) and describes the cause of the error but the message is not [visible][].
+
+```html
+<script>
+	function processForm() {
+		document.getElementById('error').innerText = ''
+		var age = document.getElementById('age').value
+		console.log(isNaN(age))
+		if (!age || isNaN(age)) {
+			document.getElementById('error').innerText = 'Age must be a number'
+		} else if (age < 1) {
+			document.getElementById('error').innerText = 'Age must be at least 1'
+		}
+	}
+</script>
+
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<span id="error" style="position: absolute; top: -9999px; left: -9999px;"></span><br />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+#### Failed Example 5
+
+The `input` element has a [form field error indicator][] that identifies it (by referencing its label) and describes the cause of the error but the message is not [included in the accessibility tree][].
+
+```html
+<script>
+	function processForm() {
+		document.getElementById('error').innerText = ''
+		var age = document.getElementById('age').value
+		console.log(isNaN(age))
+		if (!age || isNaN(age)) {
+			document.getElementById('error').innerText = 'Age must be a number'
+		} else if (age < 1) {
+			document.getElementById('error').innerText = 'Age must be at least 1'
+		}
+	}
+</script>
+
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<span id="error" aria-hidden="true"></span><br />
+	<input type="button" value="Submit" onclick="processForm()" />
+</form>
+```
+
+### Inapplicable
```

Comment:
Include one where there's a form field without an error indicator

Reply:
Having an error indicator is not part of the applicability (and it cannot be since the definition is not objective), so form fields without error indicators are still applicable.